3.6/release-notes-new-features36.md

```diff
@@ -241,6 +234,13 @@ For the ArangoSearch View case the optimization is possible if and only if:
 - all attribute accesses can be covered by stored attributes
```

Comment:
The View needs to "own" the data, right? As in, the data on disk and any regular collection indexes are not usable for ArangoSearch, correct?
```suggestion
- all attribute accesses can be covered by attributes stored in the View index
```